tutorials/nextjs-app-serverless-functions-sqldb/index.mdx

```diff
@@ -0,0 +1,948 @@
+
+    ```sh
+    scw info 
+    ```
+
+    This command displays your access key and secret key as the last two lines. The `ORIGIN` column displays `env (SCW_ACCESS_KEY)` and `env (SCW_SECRET_KEY)`, and not `default profile`.
+      
+    ```bash
+    KEY                      VALUE                                       ORIGIN
+    (...)
+    access_key               SCWF9DETBF829TAFB1TB                        env (SCW_ACCESS_KEY)
+    secret_key               9a1bcf92-xxxx-xxxx-xxxx-xxxxxxxxxxxx        env (SCW_SECRET_KEY)
+    ```
+
+3. Access the folder where you want to store your project repository and run the following command to initialize a blog template application:
+    ```bash
+    npx create-next-app --example blog-starter my-nextjs-blog
+    ```
+
+4. Type `y` when prompted to install the dependencies:
+    ```bash
+    Need to install the following packages:
+    create-next-app@14.0.3
+    Ok to proceed? (y) y
+    ```
+    This will create a folder named `my-nextjs-blog`. 
+    
+5. Access the folder you just created:
+    ```bash
+    cd my-nextjs-blog
+    ```
+
+6. Make sure you can run the application locally:
+    ```bash
+    npm run dev
+    ```
+
+7. Go to [localhost:3000](http://localhost:3000) in a browser. The blog template displays.
+
+### Creating an image containing your Next.js app
+
+1. Run the following command to create a `Dockerfile`:
+    ```bash
+    touch Dockerfile
+    ```
+2. Add the following code to it:
+    ```bash
+    # syntax=docker/dockerfile:1
+
+    FROM node:20-alpine
+      
+    #Built time arguments used for pre-rendering
+    ARG PGHOST
+    ARG PGPORT
+    ARG PGDATABASE
+    ARG PGUSER
+    ARG PGPASSWORD
+
+    WORKDIR /usr/app
+    COPY ./ ./
+    RUN npm install
+    RUN npm run build
+      
+    #Web application configuration
+    ENV PORT=8080
+      
+    #Database configuration used for dynamically rendered data. These default values should be overwritten by container runtime environment variables
+    ENV PGHOST=localhost
+    ENV PGPORT=5432
+    ENV PGDATABASE=database
+    ENV PGUSER=user
+    ENV PGPASSWORD=password
+      
+    CMD npm run start
+    ```
+
+3. Create a `.dockerignore` file by copying the `.gitignore` file:
+    ```bash
+    cp .gitignore .dockerignore
+    ```
+
+    Docker will now ignore node dependencies and build files generated during the container build.
+
+4. Build your application container:
+    ```bash
+    docker build -t my-nextjs-blog .   
+    ```
+
+5. Check that your container runs locally:
+    ```bash
+    docker run -it -p 8080:8080 my-nextjs-blog
+    ```
+
+6. Go to [localhots:8080](http://localhost:8080) in your browser. The blog template displays.
+
+    <Message type="tip">
+    When connecting to the webpage, your terminal running docker might display the following error `Error: ENOENT: no such file or directory, open '/usr/app/_posts/%5Bslug%5D.md'`. This will be fixed in further steps.
```

Comment:
```suggestion
    When connecting to the webpage, your terminal running Docker might display the following error `Error: ENOENT: no such file or directory, open '/usr/app/_posts/%5Bslug%5D.md'`. This will be fixed in further steps.
```